hooks/README.md

```diff
@@ -104,6 +120,104 @@ const MyComponent = () => {
 };
 ```
 
```

Comment:
For now, could we drop the text from `Web Platform APIs` to just before `Browser support`? (let's keep browser support as it is). My reasoning is that I think this might be a little long for the README right now. We can reconsider later however.

Reply:
Yes, I'll clean up.
Actually I make everything too big and too long. :)